ros2/naive/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.8)
```

Comment:
Let's require Ubuntu 3.16 or newer since 3.16 is what Ubuntu 20 requires and no one is likely to use an older version. Raising this value opts us into newer, better default behavior.

Reply:
> Ubuntu 3.16

?

Reply:
Sorry I meant CMake 3.16

Reply:
It would not be okay. I didn't think we still used Ubuntu 18 anywhere. In that case, raise the requirement to 3.10 since that's what Ubuntu 18 ships with. Hopefully soon we drop 18.04 because there are so many great CMake improvements between versions 3.10 and 3.16.